src/App.jsx

```diff
@@ -19,7 +28,14 @@ function App() {
     <CustomRoutes noLayout>
       <Route path="/set-password" element={<SetPasswordPage/>} />
     </CustomRoutes>
-    <Resource name="profiles" list={ListGuesser} edit={EditGuesser}/>
+    <Resource name="profiles" list={ProfileList} show={ProfileShow} edit={ProfileEdit} create={ProfileCreate} icon={Group} recordRepresentation={(record) => `${record.username}`} />
```

Comment:
oh I like username, but I don't think we should use it. It's not unique :( 
So probably email would be better

Reply:
yes that's right forgot that